src/HtmlElement/HtmlElement.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const HtmlElement = ({ element, children, ...props }) =>
+  React.createElement(element, props, children);
+
+HtmlElement.propTypes = {
+  children: PropTypes.oneOfType([
+    PropTypes.node,
+    PropTypes.arrayOf(PropTypes.node)
+  ]),
+  element: PropTypes.string
+};
+
+HtmlElement.defaultProps = {
+  children: undefined,
```

Comment:
Is this the appropriate way to say that if children are present they must be a node or nodes?

Reply:
Does PropTypes end up complaining if the default is used and children is undefined? If it does, then we should probably add `undefined` to the `oneOfType` to alleviate that.

Reply:
It does not seem to complain